includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -188,11 +188,11 @@ protected function get_permissions_check( $request, $action ) {
 				break;
 			case 'delete':
 			case 'edit':
-				$post           = get_post( (int) $request['order_id'] );
+				$post           = wc_get_order( (int) $request['order_id'] );
```

Comment:
Here we should probably rename the variable. WDYT?

Reply:
done!